### ISSUE TITLE
[Bug] Case headers dissapear when tag is very long

### ISSUE THREAD

**Issue by torsolaso** [TheHive-Project/TheHive#1992]
### Request Type
Bug

### Work Environment

| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| TheHive version / git hash | 4.1.4           |

### Problem Description

When a freeTag is very long you can see case headers:

### Steps

Create a alert and see this:
![image](https://user-images.githubusercontent.com/52453216/115536900-8bdb4b00-a29a-11eb-92b0-d40916f3dc15.png)

Create a veeery long tags and see UI
![image](https://user-images.githubusercontent.com/52453216/115537058-ba592600-a29a-11eb-81f4-944fdd852c8b.png)

### Posible Solution

- Truncate tag
- Jump to new line
- Fix tag length

**Comment by jkb-s** [comment]
If you use `CTRL`+`-` they will show up again.  
I dont think its a problem. Tags are not ment to be `that` long. Breaking them after X characters is not the proper solution because I could have ultra wide monitor and see even long ones just fine. 
  
IMHO the realy important question is: do you have a use case? What do you place in the tag that it gets so long it breaks the UI?

**Comment by torsolaso** [comment]
No,  i havent got a use case. I only detect it during my tests and i want to share it with the project. I know it is not important. But maybe there could be a easy way to put it "long...." in order to maintain the order. If there were not, I do not think it would be a problem
